lib/vagrant-google/action/read_ssh_info.rb

```diff
@@ -42,11 +47,19 @@ def read_ssh_info(google, machine)
             return nil
           end
 
-          # Read SSH network info
-          return {
-            :host => server.public_ip_address,
-            :port => 22
-          }
+          if use_private_ip then
```

Comment:
@Makdaam - not a problem at all. That's great! Always glad to see new people contributing :+1: 

To provide some details on my previous comment then, `rake lint` runs our linter based on rubocop  gem, that checks the code against the [community style guide](https://github.com/bbatsov/ruby-style-guide)

If you have any questions at all - let me know, happy to help! :smile: